src/azure-cli/azure/cli/command_modules/storage/operations/logging.py

```diff
@@ -28,8 +28,9 @@ def get_logging(client, timeout=None):
             # pylint: disable = no-member
             if ex.args and isinstance(ex.args, tuple) and hasattr(ex.args[0], 'args') and ex.args[0].args \
                     and 'Max retries exceeded with url: /?restype=service&comp=properties' in ex.args[0].args[0]:
-                raise CLIError("Your storage account doesn't support logging for {} service. Please change value for "
-                               "--services in your commands.".format(s.name))
+                raise CLIError("ConnectionError: Please check the storage account existence and credential. "
```

Comment:
>and credential [](start = 92, length = 14)

what does this mean?

Reply:
It is used to hint user to check the input storage account name and credential. If the input parameters are with invalid storage account name or credential, ConnectionError will be returned.